src/main/java/persistence/core/EntityMap.java

```diff
@@ -0,0 +1,10 @@
+package persistence.core;
+
+import java.util.HashMap;
+import java.util.Map;
+import persistence.entity.metadata.EntityMetadata;
+
+public class EntityMap {
+    protected static final Map<Class, EntityMetadata> entities = new HashMap<Class, EntityMetadata>();
```

Comment:
```suggestion
    protected static final Map<Class<?>, EntityMetadata> ENTITIES = new HashMap<>();
Comment
```
와일드카드 활용과 제네릭 타입생략 그리고 상수의 경우 변수명을 대문자로 사용하시길 제안드립니다.
포맷팅에 관한 내용은 [구글 자바 스타일가이드](https://google.github.io/styleguide/javaguide.html) 를 참고해 주시면 좋을것 같아요 🙂